momentumopt/config/solo_jump.yaml

```diff
@@ -0,0 +1,407 @@
+initial_robot_configuration:
```

Comment:
This file name should solo8 jump.
The `solo` alone name is confusing for people now...